presto-main/src/main/java/com/facebook/presto/catalogserver/CatalogServer.java

```diff
@@ -47,6 +53,16 @@
     private final SessionPropertyManager sessionPropertyManager;
     private final TransactionManager transactionManager;
     private final ObjectMapper objectMapper;
+    private final LoadingCache<CacheKey, Boolean> catalogExistsCache;
```

Comment:
Let's insert a black line before the caching related variables.

Reply:
done